api/envoy/config/cluster/v3/cluster.proto

```diff
@@ -1151,6 +1151,22 @@ message Cluster {
   // from the LRS stream here.]
   core.v3.ConfigSource lrs_server = 42;
 
+  // [#not-implemented-hide:]
+  // A list of metric names from ORCA load reports to propagate to LRS.
+  //
+  // For map fields in the ORCA proto, the string will be of the form ``<map_field_name>.<map_key>``.
+  // For example, the string ``named_metrics.foo`` will mean to look for the key ``foo`` in the ORCA
+  // ``named_metrics`` field.
+  //
+  // The special map key ``*`` means to report all entries in the map (e.g., ``named_metrics.*`` means to
+  // report all entries in the ORCA named_metrics field). Note that this should be used only with trusted
+  // backends.
+  //
+  // The metric names in LRS will follow the same semantics as this field. In other words, if this field
+  // contains ``named_metrics.foo``, then the LRS load report will include the data with that same string
+  // as the key.
+  repeated string lrs_report_endpoint_metric = 57;
```

Comment:
Note that this isn't a plural, but all the other repeated fields I'm seeing adjacent to this PR are plural.  Not sure if this kind of consistency is important to Envoy maintainers, just calling it out.

Reply:
Hi, @markdroth is this field used in gRPC or not? If it haven't be released in gRPC, I think we can re-evaluate/update it.

Reply:
We haven't started using it yet.  I'm fine with changing the name to be plural.

Reply:
I sent #35284 to make this change.